### PR TITLE
Finishing Readability Improvement.

### PR DESCRIPTION
**Pull request by Carmezim**
Done. :)
Again, I hope nothing was broken by some typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1246,52 +1245,44 @@ You'll get more graph practice in Skiena's book (see Books section below) and th
...
-        - http://www.amazon.com/Algorithm-Design-Manual-Steven-Skiena/dp/1849967202
+        - [Algorithm Design Manual](http://www.amazon.com/Algorithm-Design-Manual-Steven-Skiena/dp/1849967202)
     - Half.com is a great resource for textbooks at good prices.
     - Answers:
-        - http://www.algorithm.cs.sunysb.edu/algowiki/index.php/The_Algorithms_Design_Manual_(Second_Edition)
-        - http://blog.panictank.net/category/algorithmndesignmanualsolutions/page/2/
-    - Errata:
-        - http://www3.cs.stonybrook.edu/~skiena/algorist/book/errata
+        - [Solutions](http://www.algorithm.cs.sunysb.edu/algowiki/index.php/The_Algorithms_Design_Manual_(Second_Edition))
+        - [Solutions](http://blog.panictank.net/category/algorithmndesignmanualsolutions/page/2/))
```
Comment (jwasham) [inline README.md]: Double )) on the end

Comment (Carmezim) [inline README.md]: My bad. I am fixing it now along a few other I noticed.

**Comment by jwasham** [comment]
Thanks for the effort to make this so much more readable! Amazing!

**Comment by Carmezim** [comment]
I am happy to have done it and thank you for allowing me to contribute.

**Pull request merged**